tutorials/nnvm/deploy_ssd.py

```diff
@@ -0,0 +1,155 @@
+"""
+Deploy Single Shot Multibox Detector(SSD) model
+===============================================
+**Author**: `Yao Wang <https://github.com/kevinthesun>`_
+
+This article is an introductory tutorial to deploy SSD models with TVM.
+We will use mxnet pretrained SSD model with Resnet50 as body network and
+convert it to NNVM graph.
+"""
+import os
+import urllib
+import zipfile
+import tvm
+import mxnet as mx
+import cv2
+import numpy as np
+
+from nnvm import compiler
+from nnvm.frontend import from_mxnet
+from tvm.contrib import graph_runtime
+from mxnet.model import load_checkpoint
+
+
+######################################################################
+# Set the parameters here.
+
+model_name = "ssd_resnet50_512"
+model_file = "%s.zip" % model_name
+test_image = "dog.jpg"
+target = "llvm"
+dshape = (1, 3, 512, 512)
+dtype = "float32"
+ctx = tvm.cpu()
+
+def download(url, path, overwrite=False):
```

Comment:
move this to nnvm.testing.download, since this is not informative, take a look at other tutorials and consolidate this